src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py

```diff
@@ -174,8 +175,9 @@ def __queryForPlot(self):
         res = self.__parseFormParams()
         return MonitoringClient().generateDelayedPlot(*res["Value"]) if res["OK"] else res
 
-    def web_getPlotImg(self, fileName=None):
+    def web_getPlotImg(self, file=None):
```

Comment:
why not "fileName"?

Reply:
I believe the name was the root of the error